packages/xdl/src/detach/IosAssetArchive.ts

```diff
@@ -41,11 +41,11 @@ async function buildAssetArchiveAsync(
     path.join(intermediatesDirectory, 'Images.xcassets', 'AppIcon.appiconset')
   );
 
-  const sdkMajorVersion = parseSdkMajorVersion(context.data.manifest.sdkVersion);
+  const sdkMajorVersion = parseSdkMajorVersion(data.manifest.sdkVersion);
   const deploymentTarget = sdkMajorVersion > 30 ? '10.0' : '9.0'; // SDK31 drops support for iOS 9.0
 
   // compile asset archive
-  let xcrunargs = [].concat(
+  let xcrunargs = ([] as string[]).concat(
```

Comment:
This looks super weird to me. Maybe the best option would be to not use `.concat` and just have the final array defined explicitly? 
```
const xcrunargs = [
  'actool',
  '--minimum-deployment-target', deploymentTarget,
  'platform', 'iphoneos',
  ...
];
```
What do you think?

Reply:
++, could use // prettier-ignore to keep the formatting here

Reply:
Good call!